src/redis-check-aof.c

```diff
@@ -146,156 +266,412 @@ int readArgc(FILE *fp, long *target) {
     return readLong(fp,'*',target);
 }
 
-off_t process(FILE *fp) {
+int processRESP(FILE *fp, char *filename, int *multi) {
```

Comment:
let's denote that `multi` is an out variable.
either name `out_multi`, or add some top comment about the args and return value.

p.s. overall this file is / was in a bad shape, please add some comments on function and big code blocks explaining what they do

Reply:
OK